src/agent.rs

```diff
@@ -48,14 +48,22 @@ pub enum DecisionRule {
 /// An agent in the simulation
 #[derive(Debug, Deserialize, PartialEq, Clone)]
 pub struct Agent {
+    /// the acronym for the agent.
```

Comment:
Not sure what the document says, but these don't have to be acronyms:

```suggestion
    /// A unique identifier for the agent.
```